WordPressKit/SiteDesignServiceRemote.swift

```diff
@@ -1,35 +1,33 @@
 import Foundation
 import WordPressShared
 
-public struct SiteDesignRequest: Encodable {
-    public let previewSize: CGSize?
-    public let scale: CGFloat?
-
-    public init(previewSize: CGSize? = nil, scale: CGFloat? = nil) {
-        self.previewSize = previewSize
-        self.scale = scale
+public struct SiteDesignRequest {
+    public enum TemplateGroup: String {
+        case original
+        case singlePage = "single-page"
```

Comment:
I like your approach of having the available groups in an `enum` 👍 
I followed a different approach [on Android using just a string array](https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/1890/files#diff-3be4f77b2c2e670769618b7d0e91b4c7f77115c4cbfa763a5041e63d2ec1395bR94) but using an `enum` seems more bullet proof and I may correct this on my side. Tbh I'm not sure how often there would be a [need to add new groups](https://github.com/wordpress-mobile/gutenberg-mobile/issues/3125#issuecomment-785522260).

Reply:
I like the enum approach as well 👍

Reply:
This has been updated to use `stable`. :+1: